includes/Core/Assets/Assets.php

```diff
@@ -584,7 +584,7 @@ private function get_assets() {
 			new Stylesheet(
 				'googlesitekit-admin-css',
 				array(
-					'src'          => $base_url . 'css/admin.css',
+					'src'          => $base_url . 'css/googlesitekit-admin-css.css',
```

Comment:
These were changed recently and were not updated here. These URLs aren't really used anymore anyways so long as the asset is present in the manifest, but good to keep them pointing to the right files 😄